### PR TITLE
V8.1.0 beta

### PR DESCRIPTION
**Pull request by mcardosos**
PTAL!
@jhendrixMSFT @marstr 
For keyvault data plane, @BrucePerlerMS @pomortaz

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,29 @@
...
+### ARM
+| api                                 | version            | note                               |
+|:------------------------------------|:-------------------|:-----------------------------------|
+| arm/apimanagement                   | 2016-07-07         | new                                |
+| arm/apideployment                   | 2016-07-07         | new                                |
+| arm/billing                         | 2017-02-27-preview | new                                |
+| arm/compute                         | 2016-04-30-preview | update to latest swagger           |
+| arm/containerservice                | 2017-01-31         | update to latest swagger           |
+| arm/customer-insights               | 2017-01-01         | new                                |
+| arm/graphrbac                       | 1.6                | new                                |
```
Comment (colemickens) [inline CHANGELOG.md]: Does this mean there is composite swagger now? did I miss it?

Comment (mcardosos) [inline CHANGELOG.md]: Composite swaggers are not there yet. This is just partial support.

Comment (colemickens) [inline CHANGELOG.md]: (I need to use it in the next few days! :smile:)

Comment (mcardosos) [inline CHANGELOG.md]: Currently we can gen comp swaggers, but the generated code is not usable.
I just gen one of the swaggers. It should work just as the other parts of the SDK.

**Pull request merged**